README.md

```diff
@@ -21,7 +21,11 @@ The simplest way to get started is to install our templates with `dotnet new` us
 dotnet new --install MSBuild.Sdk.SqlProj.Templates
 ```
 
-You can then create a new project file using the following command. If you don't want to target the latest version of SQL Server you can specify a version to target using the `-s Sql<version>` switch. Please refer to [this document](https://docs.microsoft.com/dotnet/api/microsoft.sqlserver.dac.model.sqlserverversion) for the available versions.
+You can then create a new project file using the following command. If you don't want to target the latest version of SQL Server you can specify a version to target using the `-s Sql<version>` switch. 
+
+See [How to determine the version, edition, and update level of SQL Server and its components](https://support.microsoft.com/en-us/help/321185/how-to-determine-the-version-edition-and-update-level-of-sql-server-an) to map from the SQL Server year to the version number, e.g. `SQL Server 2019` to `15.0`.
```

Comment:
"to map from the SQL Server SKU name to the version number,"
(as Azure and AzureDW are also relevant)